automatic/screencloud/screencloud.nuspec

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>screencloud</id>
+    <version>1.3.0</version>
+    <packageSourceUrl>https://github.com/p3lim/chocolatey-packages/tree/master/screencloud</packageSourceUrl>
```

Comment:
Incorrect `packageSourceUrl`, please update it to `https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/screencloud`